lib/aytests/certs_factory.rb

```diff
@@ -0,0 +1,71 @@
+module AYTests
+  # Certificates factory
+  #
+  # Generates a certificate (with its corresponding key).
+  class CertsFactory
+    # @return [OpenSSL::X509::Certificate] CA certificate
+    attr_reader :ca_crt
+    # @return [OpenSSL::PKey::RSA] CA key
+    attr_reader :ca_key
+
+    # Highest serial number
+    SERIAL_LIMIT = 9999
+    # Certificates expiration time (in seconds)
+    EXPIRATION   = 3600
+
+    # Constructor
+    #
+    # @param ca_cert_ path [Pathname] Path to CA certificate
+    # @param ca_key_path   [Pathname] Path to CA certificate
+    def initialize(ca_crt_path:, ca_key_path:)
+      @ca_crt = OpenSSL::X509::Certificate.new(ca_crt_path.read)
```

Comment:
Do we have to add any additional requirements (gem installation while setup the test environment, documentation) here ?